cli/cmd/profile.go

```diff
@@ -78,7 +76,7 @@ Example:
 				if options.openAPI != "" {
 					return errors.New("You must specify exactly one of --template or --open-api")
 				}
-				return renderProfileTemplate(buildConfig(options.namespace, options.name), os.Stdout)
+				return profiles.RenderProfileTemplate(options.namespace, args[0], controlPlaneNamespace, os.Stdout)
```

Comment:
TIOLI -- the `options.name` field was added in #1867, so you can use that here instead of `args[0]`.